examples/examples_heterogeneous/002_heterogeneous_using_wind_data.py

```diff
@@ -75,26 +77,45 @@
 # Get the power output of the turbines
 turbine_powers = fmodel.get_turbine_powers() / 1000.0
 
-# Now repeat using the wind_data object and heterogeneous_inflow_config_by_wd
+# Now repeat using the wind_data object and HeterogeneousMap object
 # First, create the speedups for the two wind directions
 speed_ups = [[1.0, 1.25, 1.0, 1.25], [1.0, 1.35, 1.0, 1.35]]
 
-# Create the heterogeneous_inflow_config_by_wd dictionary
+# Now define the HeterogeneousMap object
+heterogeneous_map = HeterogeneousMap(
+    x = x_locs,
+    y = y_locs,
+    speed_multipliers = speed_ups,
+    wind_directions = [270.0, 280.0],
+)
+
+# Now create a new TimeSeries object including the heterogeneous_inflow_config_by_wd
+time_series = TimeSeries(
```

Comment:
I'm not quite sure what is going on here, because it looks like the examples were run to completion in your last commit, and I don't think I touched anything that would have broken this?

Reply:
I think it's because you changed the name from speedups to speedmultipliers for the by wd types but then refererred back to speedups when declaring the heterogeneous_inflow_config_by_wd, fixing now

Reply:
@paulf81 a shoot, sorry, my bad!